src/svgdevicecontext.cpp

```diff
@@ -240,6 +240,12 @@ void SvgDeviceContext::StartPage()
     // Initialize the flag to false because we want to know if the font needs to be included in the SVG
     m_vrvTextFont = false;
 
+    // default styles
+    m_currentNode = m_currentNode.append_child("style");
+    m_currentNode.append_attribute("type") = "text/css";
+    m_currentNode.append_child(pugi::node_pcdata).set_value("g.page-margin{font-family:Times;} g.tempo{font-weight:bold;} g.dir, g.dynam {font-style:italic;}");
```

Comment:
Concerning @ahankinson comment about it: it would indeed be better, but having all of them here is already better than before. One step forward. Eventually, we should pass a `Style` object to `StartPage` and get them for there. I am fine leaving this like this for now.